stock_reception_screen_measuring_device/models/stock_reception_screen.py

```diff
@@ -33,6 +38,22 @@ def _compute_scan_requested(self):
                 and record.product_packaging_id.measuring_device_id
             )
 
+    @api.depends(
+        "product_packaging_id.lngth",
+        "product_packaging_id.width",
+        "product_packaging_id.height",
+    )
+    def _compute_package_dimensions(self):
+        for record in self:
+            pack = record.product_packaging_id
+            dimension_values = [pack.lngth, pack.height, pack.width]
+            if all(dimension_values):
+                record.display_package_dimensions = " x ".join(
+                    [str(val) for val in dimension_values]
```

Comment:
Add mm unit would be nice
```suggestion
                    [f"{str(val)}mm" for val in dimension_values]
```